Cargo.toml

```diff
@@ -1,19 +1,20 @@
 [package]
 name = "jsonwebtoken"
-version = "6.0.1"
+version = "7.0.0"
 authors = ["Vincent Prouillet <prouillet.vincent@gmail.com>"]
 license = "MIT"
 readme = "README.md"
 description = "Create and parse JWT in a strongly typed way."
 homepage = "https://github.com/Keats/rust-jwt"
 repository = "https://github.com/Keats/rust-jwt"
 keywords = ["jwt", "web", "api", "token", "json"]
+edition = "2018"
 
 [dependencies]
 serde_json = "1.0"
 serde_derive = "1.0"
 serde = "1.0"
-ring = "0.14.4"
+ring = "0.14.6"
```

Comment:
Rustls 0.16 was released, which uses *ring* 0.16. Trust-DNS should have a release soon, both of which does the same. So, it would be great to use *ring* 0.16 here too. I am happy to help with this.